eq-author/src/App/page/Design/PageHeader.js

```diff
@@ -0,0 +1,182 @@
+import React from "react";
+import IconButtonDelete from "components/buttons/IconButtonDelete";
+import DeleteConfirmDialog from "components/DeleteConfirmDialog";
+import iconPage from "./icon-dialog-page.svg";
+import { isFunction, flowRight } from "lodash";
+import PropTypes from "prop-types";
+import { propType } from "graphql-anywhere";
+import gql from "graphql-tag";
+
+import Button from "components/buttons/Button";
+import IconText from "components/IconText";
+import DuplicateButton from "components/buttons/DuplicateButton";
+import { Label } from "components/Forms";
+import VisuallyHidden from "components/VisuallyHidden";
+import AliasEditor from "components/AliasEditor";
+
+import withMovePage from "./withMovePage";
+import withDeletePage from "./withDeletePage";
+import withDuplicatePage from "./withDuplicatePage";
+import MovePageQuery from "./MovePageModal/MovePageQuery";
+import { Toolbar, Buttons } from "./EditorToolbar";
+import IconMove from "./EditorToolbar/icon-move.svg?inline";
+import MovePageModal from "./MovePageModal";
+
+export class PageHeader extends React.Component {
+  state = {
+    showDeleteConfirmDialog: false,
+    showMovePageDialog: false,
+  };
+
+  handleDuplicatePage = e => {
+    e.preventDefault();
+    const { match, onDuplicatePage, page } = this.props;
+    onDuplicatePage({
+      sectionId: match.params.sectionId,
+      pageId: page.id,
+      position: page.position + 1,
+    });
+  };
+  handleOpenDeleteConfirmDialog = () =>
+    this.setState({ showDeleteConfirmDialog: true });
+
+  handleCloseDeleteConfirmDialog = cb =>
+    this.setState(
+      { showDeleteConfirmDialog: false },
+      isFunction(cb) ? cb : null
+    );
+
+  handleDeletePageConfirm = () => {
+    const { onDeletePage, page } = this.props;
+    this.handleCloseDeleteConfirmDialog(() => onDeletePage(page));
+  };
+
+  handleOpenMovePageDialog = () => {
+    this.setState({ showMovePageDialog: true });
+  };
+
+  handleCloseMovePageDialog = cb => {
+    this.setState({ showMovePageDialog: false }, isFunction(cb) ? cb : null);
+  };
+
+  handleMovePage = args => {
+    this.handleCloseMovePageDialog(() => this.props.onMovePage(args));
+  };
+
+  renderMovePageModal = ({ loading, error, data }) => {
+    const { page } = this.props;
+
+    if (loading || error) {
+      return null;
+    }
+
+    return (
+      <MovePageModal
+        questionnaire={data.questionnaire}
+        isOpen={this.state.showMovePageDialog}
+        onClose={this.handleCloseMovePageDialog}
+        onMovePage={this.handleMovePage}
+        sectionId={page.section.id}
+        page={page}
+      />
+    );
+  };
+
+  render() {
+    const {
+      page,
+      onChange,
+      onUpdate,
+      match,
+      moveDisabled,
```

Comment:
Could we prefix this and the one below with `is` so that its obvious that they are flags and not a way of moving the disabled button.